docs/source/features/fp8.rst

```diff
@@ -0,0 +1,40 @@
+FP8 usage
```

Comment:
@ksivaman  can you copy paste this change to https://github.com/NVIDIA/NeMo/blob/main/docs/source/features/mixed_precision.rst